osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/jms/MessageType.java

```diff
@@ -98,6 +98,7 @@
   SET_TRANSITION,
   START_SELF_TEST,
   STOP_SELF_TEST,
+  SYSTEM_EVENT,
```

Comment:
Comment at the top of this class requests alphabetical order. To keep it that way, it looks like the SYSTEM_EVENT had better be placed between the lines for SYNCHRONIZE_TIME and UPDATE_DEVICE_CDMA_SETTINGS.

Reply:
moved value to correct place

Reply:
"moved value to correct place" ->
Was this more of an intention than something actually done, or did you just forget to push the change?

Reply:
Moved